src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -187,6 +187,10 @@ @interface MTRDevice ()
 
 @end
 
+@protocol MTRDeviceUnitTestDelegate <MTRDeviceDelegate>
+- (void)unitTestReportEndForDevice:(MTRDevice *)device;
```

Comment:
I assume this protocol declaration is needed just so `@selector(unitTestReportEndForDevice:)` doesn't error out?  Might be worth documenting that.

Reply:
Done.